apiserver/params/internal.go

```diff
@@ -270,8 +277,11 @@ type SetUnitStateArgs struct {
 // SetUnitStateArg holds a unit tag and a map with KV-pairs that represent a local
```

Comment:
For future reference, can you add a comment that these fields are optional and deletions can be specified by the caller by setting the pointer to an empty value?

Reply:
added